libcob/common.c

```diff
@@ -2294,6 +2294,13 @@ cob_stop_run (const int status)
 	exit (status);
 }
 
+void
+cob_stop_error (void)
+{
+	cob_runtime_error ("STOP ERROR");
+	cob_stop_run(1);
```

Comment:
should use `cob_hard_failure ()` now instead of `cob_stop_run`

Reply:
Done (by Nicolas)